src/modules/organizations/containers/details-form-container.jsx

```diff
@@ -68,12 +68,14 @@ class DetailsFormContainer extends React.Component {
         <h5>Details</h5>
         <FormContainer onSubmit={this.handleSubmit} onReset={this.resetOrganization}>
           <fieldset className="form__fieldset">
-            <DisplayNameSlugEditor
-              origin={config.zooniverseURL}
-              resource={organization}
-              resourceType="organization"
-              ref={(node) => { this.fields.display_name = node; }}
-            />
+            <label className="form__label" htmlFor="display_name">
```

Comment:
`htmlFor` isn't necessary when the label wraps the input. It's used to link a label and input when they aren't implicitly connected in the markup.

The a11y linter might warn about it being missing, but you can ignore than warning in cases like this.